build.gradle

```diff
@@ -1,5 +1,5 @@
 plugins {
-    id 'eu.xenit.docker-compose' version '5.3.2' apply false
+    id 'eu.xenit.docker-compose' version '5.5.0' apply false
```

Comment:
docker-compose files have to be upgrade to V3.x

version: '3.2' (Found in docker.yml files). This comment can be ignored.